docs/property-grid.md

```diff
@@ -0,0 +1,418 @@
+| `"itemvalues"` | Customized text inputs for entering value-text pairs | Use this type for arrays of objects with the following structure: `{ value: any, text: string }`. For example, Dropdown, Checkbox, and Radiogroup questions use this type for the [`choices`](https://surveyjs.io/Documentation/Library?id=QuestionSelectBase#choices) property. |
+| `"value"` | Button that opens a dialog window | The dialog window displays the survey element and allows users to set the element's default value. |
+
+You can add the type to the `name` property after a colon character as a shortcut:
+
+```js
+Survey.Serializer.addProperty("question", 
+  { name: "my-boolean-property", type: "boolean" }
+  // ===== or =====
+  { name: "my-boolean-property:boolean" }
+);
+```
+
+#### `default`
+
+A default value for the property. If not specified, `default` equals to `""` for string values, 0 for numbers, `false` for Boolean values. The default value is not serialized into survey JSON definition.
+
+```js
+Survey.Serializer.addProperty("dropdown", 
+  { name: "my-string-property", default: "custom-default-value" }
+);
+
+Survey.Serializer.addProperty("checkbox", 
+  { name: "my-numeric-property", type: "number", default: 100 }
+);
+
+Survey.Serializer.addProperty("question", 
+  { name: "my-boolean-property", type: "boolean", default: true }
+);
+```
+#### `displayName`
+
+A string value that specifies a property caption. If not specified, the [`name`](#name) value is used instead.
+
+```js
+Survey.Serializer.addProperty("dropdown", 
+  { name: "my-string-property", displayName: "Custom Caption" }
+);
+```
+
+#### `choices`
+
+An array of selection choices or a function that loads the choices from a web service. Applies only to text and numeric properties. If `choices` are specified, Survey Creator renders a drop-down menu as the property editor.
+
+```js
+// Define `choices` locally
+Survey.Serializer.addProperty("question", {
+  name: "my-string-property",
+  choices: [ "option1", "option2", "option3" ],
+  // If item captions should be different from item values:
+  // choices: [
+  //   { value: "option1", text: "Option 1" },
+  //   { value: "option2", text: "Option 2" },
+  //   { value: "option3", text: "Option 3" },
+  // ],
+  default: "option1"
+});
+
+// Load `choices` from a web service
+Survey.Serializer.addProperty("survey", {
+  name: "country",
+  category: "general",
+  choices: function (obj, choicesCallback) {
+    const xhr = new XMLHttpRequest();
+    xhr.open("GET", "https://surveyjs.io/api/CountriesExample");
+    xhr.setRequestHeader( "Content-Type", "application/x-www-form-urlencoded");
+    xhr.onload = function () {
+      if (xhr.status === 200) {
+        const response = JSON.parse(xhr.response);
+        const result = [];
+        // Make the property nullable
+        result.push({ value: null });
+        // Web service returns objects that we convert to the `{ value, text }` format
```

Comment:
Is it necessary to say "we"? For example, can we say the following?

`// Web service returns objects that are converted to the `{ value, text }` format`